tests/integration/test_xnat2bids.py

```diff
@@ -73,5 +73,11 @@ def test_xnat2bids():
             dataset = pydicom.dcmread(os.path.join(d, f))
             idx = series_idx(f)
             assert str(idx) not in skiplist
-            assert dataset.data_element("ProtocolName").value == series_desc
-            assert dataset.data_element("SeriesDescription").value == series_desc
+            if series_desc.__contains__("SBRef"):
+                assert dataset.data_element("ProtocolName").value == series_desc.replace(
+                    "_SBRef", ""
```

Comment:
It doesn't seem right to be modifying data with `replace` in an assertion/test. I wonder if there is better way to do this.... like if we have this information already encoded in the package we should be reusing that ... it could get out of hand to be duplicating the logic for this replacements in the package and in the tests 🤔

Reply:
@mirestrepo Currently, the test is checking the 'ProtocolName' and 'SeriesDescription' from the xnat-export/ directory. I think it would be better to point towards the bids/ directory. Then, we can make our assertions on the data_element values after Heudiconv has completed its processing (i.e. appending _sbref for diffusion scans). What are your thoughts?

Reply:
Disregard my previous idea. BIDS won't have the dicom meta-data pydicom is using here to generate the dataset.